### ISSUE TITLE
Use unique topic for each consensus group.

### ISSUE THREAD

**Issue by vldm** [stegos/stegos#323]
For now consensus use single broadcast "topic" for all consensus messages, in situation where fork allowed, we should dynamicly create and remove topics for consensus group based on group identifier.

**Comment by rtsisyk** [comment]
Blocked by #322

**Comment by echupriyanov** [comment]
Why is it blocked by #322 ? API remains still the same.

**Comment by echupriyanov** [comment]
Oops, deleted by mistake.

**Comment by echupriyanov** [comment]
Blocked by #322

**Comment by vldm** [comment]
@echupriyanov I think this task is low priority and highly related to network layer, working on this task probably will change api (introduce some kind of unscribtions), so working on it could slow or broke work on #322 .

**Comment by rtsisyk** [comment]
#322 has already been pushed.

**Comment by echupriyanov** [comment]
> #322 has already been pushed.

Thats still not enough

**Comment by rtsisyk** [comment]
Is it actual? We already handle this problem by using `height` value in the message.

**Comment by vldm** [comment]
@rtsisyk Height is like emulated unicast. It works, but on more high level.

With unique consensus group we will filter-out nodes from consensus that is not related to us.

**Comment by vldm** [comment]
For now, we using pBFT-like consensus rarely, so there no possibility of message interference between two consensus groups. 

Hacker still can spam into consensus topic, so we can research how to protect topic, but it probably out-of scope of current task @rtsisyk  initial problem was mixing messages from different groups.
